sklearn/ensemble/tests/test_forest.py

```diff
@@ -204,6 +204,23 @@ def check_probability(name):
                                   np.exp(clf.predict_log_proba(iris.data)))
 
 
+@pytest.mark.parametrize('name', FOREST_CLASSIFIERS)
+def test_warning_raised_with_deprecated_n_estimator(name):
+    print(FOREST_CLASSIFIERS)
+    ForestClassifier = FOREST_CLASSIFIERS[name]
+    clf = ForestClassifier(n_estimators=10, random_state=1, max_features=1,
+                           max_depth=1)
+    assert_warns(FutureWarning, func=clf.fit, X=iris.data, y=iris.target)
```

Comment:
Explicitly passing n_estimators=10 should not result in a warning.
Passing nothing for n_estimators should result in n warning